pages/index.js

```diff
@@ -83,6 +81,15 @@ const ExerciseCard = styled.li`
   justify-content: flex-end;
   height: 25rem;
   width: 100%;
+
+  &:hover {
+    background-color: #f0f8ff;
+    transform: scale(1.1);
+  }
+
```

Comment:
Remember that hover effects are generally not applicable on touch devices. Try to use media queries and device detection.